nsv13/code/controllers/subsystem/overmap_mode.dm

```diff
@@ -361,6 +367,45 @@ SUBSYSTEM_DEF(overmap_mode)
 	if(mode.difficulty <= 0)
 		mode.difficulty = 1
 
+/datum/controller/subsystem/overmap_mode/proc/force_mode(new_mode)
+	if(!new_mode) //You can't do something with nothing
+		return FALSE
+	if(!mode)
+		forced_mode = new_mode
+		return initial(forced_mode.name)
+	else //We have to remove the original one first
+		QDEL_NULL(mode)
+		mode = new new_mode()
+		forced_mode = mode
+		mode_initialised = FALSE //Reset this in case something fails after it.
+		setup_overmap_mode() //we will have to reinitialize manually
+		return mode.name
```

Comment:
This will still runtime because `setup_overmap_mode()` is always called on roundstart by ticker, which will cause the proc to be called again, which in turn will runtime.

Reply:
What goes wrong exactly? I literally used this and it worked on my machine

Reply:
`setup_overmap_mode()` sets up objectives, which reliably runtimes if done twice. You can test this by doing the hardmode vote before roundstart, having it succeed, then starting the round (which will runtime)